proto/axelar/nexus/exported/v1beta1/types.proto

```diff
@@ -87,3 +87,23 @@ enum TransferDirection {
   TRANSFER_DIRECTION_OUTGOING = 2
       [ (gogoproto.enumvalue_customname) = "Outgoing" ];
 }
+
+message GeneralMessage {
+  enum Status {
+    option (gogoproto.goproto_enum_prefix) = false;
+    option (gogoproto.goproto_enum_stringer) = true;
+
+    STATUS_UNSPECIFIED = 0 [ (gogoproto.enumvalue_customname) = "NonExistent" ];
+    STATUS_APPROVED = 1 [ (gogoproto.enumvalue_customname) = "Approved" ];
+    STATUS_EXECUTED = 2 [ (gogoproto.enumvalue_customname) = "Executed" ];
```

Comment:
```suggestion
    STATUS_PENDING = 1 [ (gogoproto.enumvalue_customname) = "Pending" ];
    STATUS_APPROVED = 2 [ (gogoproto.enumvalue_customname) = "Approved" ];
```
Perhaps? Execution might be lazy depending on the destination.

Reply:
Once the message got set in nexus, it should be in `Approved` state, anyone can execute from there.